erts/emulator/beam/erl_bif_persistent.c

```diff
@@ -122,6 +122,7 @@ typedef struct {
     HashTable* hash_table;
     Eterm heap[3];
     Eterm tuple;
+    Eterm old_bucket;
```

Comment:
`old_bucket` does not need to be in ErtsPersistentTermPut2Context as it's not used over traps, but it doesn't do any harm either so...

Reply:
Good point, fixed. :)